server/routes/authroutes.js

```diff
@@ -1,14 +1,19 @@
 const express = require('express');
 const userController = require('../controllers/auth');
 const router = express.Router()
+const Validate = require('../middlewares/validators')
+
+
+// const { user } = Validate
```

Comment:
commented out code can be confusing, please remove it.

Reply:
Thanks for the feedback. I'll fix that.